code-reviews/recipes/azure-pipelines-yaml.md

```diff
@@ -0,0 +1,119 @@
+# YAML(Azure Pipelines) Code Reviews
+
+## Syle Guide
+
+[CSE](../../CSE.md) developers follow the [YAML schema reference](https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema).
+
+## Code Analysis / Linting
+
+The most popular YAML linter is [YAML](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml) extension. This extension provides YAML validation, document outlining, Auto completion, Hover support and Formatter features.
+
+## VS Code Extensions
+
+There is an [Azure Pipelines for VS Code](https://marketplace.visualstudio.com/items?itemName=ms-azure-devops.azure-pipelines) extension to add syntax highlighting and autocompletion for Azure Pipelines YAML to VS Code. It also helps you set up continuous build and deployment for Azure WebApps without leaving VS Code.
+
+## YAML in Azure Pipelines Overview
+
+When the pipeline is triggered, before running the pipeline, there are some different phases such as `Queue Time`, `Compile Time` and `Runtime`, in these phases variables are interpretted by their [runtime expression syntax](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#runtime-expression-syntax).
```

Comment:
Perhaps a link to explanation of the various phases would be good here

Reply:
Sure, provided an external link, explains these phases clearly.